### PR TITLE
adds a new hat

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

Adds a new hat, it creates MORE hats!
image

## Why It's Good For The Game

image

## Changelog
:cl:
add: Adds a new hat
/:cl:

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
hmm yes

**Comment by wesoda25** [comment]
hmm yes

**Comment by MrDoomBringer** [comment]
image
the finest of diffs

**Comment by Fikou** [comment]
code\modules\clothing\head\misc.dm:525:error: pick: undefined proc
this pr is so cursed an internal byond proc is undefined

**Comment by trollbreeder** [comment]
Hmm, yes, today i will submit a clothing PR without updating master

**Comment by Farquaar** [comment]
You should give it some texture or shading. Looks kind of rough as-is.

**Comment by Qustinnus** [comment]
oranges: no more sprite bloat
also oranges: we need HATS

**Comment by OverlyGeneric** [comment]
looks blocky and unpolished.

just port hippie's hatstacking. it's fun.

**Comment by cacogen** [comment]
looks good

File: code/modules/clothing/head/misc.dm [hunk 1]
```diff
@@ -514,3 +514,13 @@
...
+/obj/item/clothing/head/more
+	name = "hat of more hats"
+	desc = "Chop chop!"
+	icon_state = "morehat"
+	item_state = "morehat"
+
+/obj/item/clothing/head/more/attack_self(mob/M)
+	var/list/hat_list = subtypesof(/obj/item/clothing/head)
+	var/hat = pick(hat_list)
+ 	new hat(get_turf(M))
```
Comment (ATH1909) [inline code/modules/clothing/head/misc.dm]: Can this summon those temporary holders that hold drones and pAIs when they're being used as hats? Can this summon ERT helmets?

*gasp*
Can this summon a... a... a-a-a CAKEHAT?!

**Comment by Fikou** [comment]
oranges didnt even post a funny comment on this pr im disappointed

**Comment by PKPenguin321** [comment]
>oranges didnt even post a funny comment on this pr im disappointed
because this isn't a joke

**Pull request closed without merging**